broker.go

```diff
@@ -949,6 +972,91 @@ func (b *Broker) sendAndReceiveSASLOAuth(provider AccessTokenProvider) error {
 	return nil
 }
 
+func (b *Broker) sendAndReceiveSASLSCRAMv1(client SCRAMClient) error {
```

Comment:
Why client argument is not used in the method?

Reply:
It should be but it's should be passed as an argument at all because I am using the client attached to the broker config. Fixed.